src/auto/about_dialog.rs

```diff
@@ -472,97 +463,97 @@ impl<O: IsA<AboutDialog>> AboutDialogExt for O {
 
 unsafe extern "C" fn activate_link_trampoline<P, F: Fn(&P, &str) -> Inhibit + 'static>(this: *mut ffi::GtkAboutDialog, uri: *mut libc::c_char, f: glib_ffi::gpointer) -> glib_ffi::gboolean
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
     f(&AboutDialog::from_glib_borrow(this).unsafe_cast(), &GString::from_glib_borrow(uri)).to_glib()
 }
 
 unsafe extern "C" fn notify_artists_trampoline<P, F: Fn(&P) + 'static>(this: *mut ffi::GtkAboutDialog, _param_spec: glib_ffi::gpointer, f: glib_ffi::gpointer)
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
     f(&AboutDialog::from_glib_borrow(this).unsafe_cast())
 }
 
 unsafe extern "C" fn notify_authors_trampoline<P, F: Fn(&P) + 'static>(this: *mut ffi::GtkAboutDialog, _param_spec: glib_ffi::gpointer, f: glib_ffi::gpointer)
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
     f(&AboutDialog::from_glib_borrow(this).unsafe_cast())
 }
 
 unsafe extern "C" fn notify_comments_trampoline<P, F: Fn(&P) + 'static>(this: *mut ffi::GtkAboutDialog, _param_spec: glib_ffi::gpointer, f: glib_ffi::gpointer)
 where P: IsA<AboutDialog> {
-    let f: &F = transmute(f);
+    let f: &F = &*(f as *const F);
```

Comment:
You probably want to remove the `transmute` in the manual code too. There are 5 cases like this in the manual code that can be changed, leaving only the cases where the trampolines are transmuted (I'm not sure why exactly this is needed)